Chinese/DesertHell/OldStory/Cat2Dog/变成狗了.md

```diff
@@ -0,0 +1,2 @@
+这只猫变成了一只狗。它很茫然。它就去找白胡子老头，白胡子老头说，说什么呢？
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.